tests/test.empty-macro-arg/expected.dylan

```diff
@@ -0,0 +1,21 @@
+module: test-empty-macro-arg
+synopsis: minimal test case for melange bug when passing empty macro args
+author: Wim Vander Schelden
+copyright: X11
```

Comment:
can you drop this line and leave it as implied the same as the rest of the code? or explicitly state that it is licensed according to the License.txt as other files do? (same wording)